project/CLI.py

```diff
@@ -92,15 +105,41 @@ def __create(self, args):
             return "Game Created"
         return "Game Failed"
 
+    def __performTeamEdit(self, args):
+        if args[1] == "name":
```

Comment:
By indexing args you run the risk of an IndexError. 
You should try except that.

Reply:
Alright its fixed in latest commit.